### ISSUE TITLE
Take folder order preference into account when order retention is enabled

### ISSUE THREAD

**Issue by Dan503** [Dan503/gulp-file-loader#8]
With a gulp task like this:

```js
gulp.task('load:scss', function () {
    var dest = local_theme_folder+'/sass';
    return gulp.src([
        local_theme_folder+'/sass/variables/**/*.scss',
        local_theme_folder+'/sass/mixins/**/*.scss',
        local_theme_folder+'/sass/components/**/*.scss',
        local_theme_folder+'/sass/pages/**/*.scss',
    ])
        .pipe(fileLoader({ preset: 'scss', dest: dest, fileName: '__includes.scss' }))
        .pipe(gulp.dest(dest))
```

The user wants the variables import to always be placed toward the top of the import file.

Currently any new variable files that are added are added to the bottom of the file which is annoying because you need to manually move them into position every time.

**Comment by Dan503** [comment]
Instead of just pasting the new import at the bottom of the file, I should try and keep the folders grouped together by analysing how close of a match each line is to the file in question and place it at the bottom of the list that is the closest match to it's path.

**Comment by Dan503** [comment]
Fixed in v2.1.0
